toolz/itertoolz.py

```diff
@@ -495,6 +496,21 @@ def concatv(*seqs):
     return concat(seqs)
 
 
+def cycles(n, seq):
+    """ Cycles through the sequence n-times.
+
+    Note: If ``n`` is ``None``, cycle infinitely.
+
+    >>> list(cycles(2, [1, 2, 3]))
+    [1, 2, 3, 1, 2, 3]
+    """
+
+    if n is None:
```

Comment:
+1 on @llllllllll 's comment.  @jakirkham are you submitting this because you actively need it for a real world application or because it might be nice.  There are probably too many "nice" functions out there to put all of them into toolz.